Makefile

```diff
@@ -1,39 +0,0 @@
-pkg = $(shell basename $(CURDIR))
-
-all: build
-
-NAMESPACE: R/*
-	Rscript -e "devtools::document()"
-
-README.md: README.Rmd
```

Comment:
@kuriwaki, it's fine if you delete the Makefile.  None of my other packages use them.  But you'll need to remember to manually knitr README.Rmd to update README.md.

That tradeoff is probably worth it to you.  I just wanted to state the new manual requirement.